cmd/eventlistenersink/main.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	triggersclientset "github.com/tektoncd/triggers/pkg/client/clientset/versioned"
+	"github.com/tektoncd/triggers/pkg/sink"
+	kubeclientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func main() {
+	log.Print("EventListener pod started")
+
+	// Get env variables
+	elName := os.Getenv("EL_NAME")
+	if elName == "" {
+		log.Fatal("EL_NAME env not found")
+	}
+	elNamespace := os.Getenv("EL_NAMESPACE")
+	if elNamespace == "" {
+		log.Fatal("EL_NAMESPACE env not found")
+	}
+	port := os.Getenv("PORT")
+	if port == "" {
+		log.Fatal("PORT env not found")
+	}
+
+	// Configure clients
+	clusterConfig, err := rest.InClusterConfig()
+	if err != nil {
+		log.Fatal("Failed to get in cluster config", err)
+	}
+	kubeClient := kubeclientset.NewForConfigOrDie(clusterConfig)
+	triggersClient := triggersclientset.NewForConfigOrDie(clusterConfig)
+
+	// Create sink Resource
+	r := sink.Resource{
+		KubeClient:             kubeClient,
+		TriggersClient:         triggersClient,
+		EventListenerName:      elName,
+		EventListenerNamespace: elNamespace,
+	}
+
+	// Listen and serve
+	log.Printf("Listen and serve on port %s", port)
+	http.HandleFunc("/", r.HandleEvent)
+	log.Fatal(http.ListenAndServe(fmt.Sprintf(":%s", port), nil))
+}
```

Comment:
i like how we've got the sink package, separate from the logic here :D

I think we could even break this up and add some unit tests around it too - even tho there's not too much here (which makes it easier to break up :D):

- function to get the configuration (env var values)
- function to get the clients
- function to pass execution to the http handler